src/grdmath.c

```diff
@@ -167,293 +167,296 @@ static void Free_Ctrl (struct GMT_CTRL *GMT, struct GRDMATH_CTRL *C) {	/* Deallo
 static int usage (struct GMTAPI_CTRL *API, int level) {
 	const char *name = gmt_show_name_and_purpose (API, THIS_MODULE_LIB, THIS_MODULE_CLASSIC_NAME, THIS_MODULE_PURPOSE);
 	if (level == GMT_MODULE_PURPOSE) return (GMT_NOERROR);
-	GMT_Message (API, GMT_TIME_NONE, "usage: %s [%s]\n\t[%s]\n\t[-D<resolution>][+f] [%s]\n\t[-M] [-N] [-S] [%s] [%s] [%s] [%s]\n\t[%s] [%s]"
-		" [%s]\n\t[%s] [%s]\n\t[%s] [%s] [%s]\n\t%s [%s]", name, GMT_Rgeo_OPT, GMT_A_OPT, GMT_I_OPT, GMT_V_OPT, GMT_a_OPT, GMT_bi_OPT, GMT_di_OPT,
+	GMT_Usage (API, 0, "usage: %s [%s] [%s] [-D<resolution>[+f] [%s] [-M] [-N] [-S] [%s] [%s] [%s] [%s] [%s] [%s] "
+		"[%s] [%s] [%s] [%s] [%s] [%s] %s [%s] A B op C op D op ... = <outgrid>\n", name, GMT_Rgeo_OPT, GMT_A_OPT, GMT_I_OPT, GMT_V_OPT, GMT_a_OPT, GMT_bi_OPT, GMT_di_OPT,
 		GMT_e_OPT, GMT_f_OPT, GMT_g_OPT, GMT_h_OPT, GMT_i_OPT, GMT_n_OPT, GMT_r_OPT, GMT_s_OPT, GMT_x_OPT, GMT_PAR_OPT);
```

Comment:
-e and -r are listed here but not in rst documentation.

Reply:
-x is not in the usage message despite GMT_x_OPT

Reply:
I am removing -e.  I see -r in the rst docs.  Where do you see it missing?  -x in usage requires OpenMP build so unless you did that you won't see -x listed.

Reply:
Sorry, another typing mistake. I meant -s, not -r.

Reply:
Thanks.  I wonder about this one: grdmath has a handful of operators that read tables, hence there is -a, -b, -d, etc.  So I think I hsould change my mind and add back -e and add -e -s to the docs to be consistent.  No reason why tables read by grdmath cannot access some of the table options.

Reply:
Well, not -s since that is for table output which is not releveant here.  So will remove -s.